zhaquirks/schneiderelectric/thermostat.py

```diff
@@ -0,0 +1,576 @@
+            access="rw",
+            is_manufacturer_specific=True,
+        )
+        se_cool_temperature_high_limit: Final = ZCLAttributeDef(
+            id=0x0022,
+            type=t.int16s,
+            access="rw",
+            is_manufacturer_specific=True,
+        )
+        se_cool_temperature_low_limit: Final = ZCLAttributeDef(
+            id=0x0023,
+            type=t.int16s,
+            access="rw",
+            is_manufacturer_specific=True,
+        )
+        se_cooling_output_mode: Final = ZCLAttributeDef(
+            id=0x0030,
+            type=SECoolingOutputMode,
+            access="rw",
+            is_manufacturer_specific=True,
+        )
+        se_heating_output_mode: Final = ZCLAttributeDef(
+            id=0x0031,
+            type=SEHeatingOutputMode,
+            access="rw",
+            is_manufacturer_specific=True,
+        )
+        se_maximum_idle_time: Final = ZCLAttributeDef(
+            id=0x0041,
+            type=t.uint16_t,
+            access="rw",
+            is_manufacturer_specific=True,
+        )
+        se_anti_idle_exercise_time: Final = ZCLAttributeDef(
+            id=0x0042,
+            type=t.uint16_t,
+            access="rw",
+            is_manufacturer_specific=True,
+        )
+        se_preffered_exercise_time: Final = ZCLAttributeDef(
+            id=0x0043,
+            type=t.uint16_t,
+            access="rw",
+            is_manufacturer_specific=True,
+        )
+        se_min_off_time: Final = ZCLAttributeDef(
+            id=0x0044,
+            type=t.uint16_t,
+            access="rw",
+            is_manufacturer_specific=True,
+        )
+        se_min_on_time: Final = ZCLAttributeDef(
+            id=0x0045,
+            type=t.uint16_t,
+            access="rw",
+            is_manufacturer_specific=True,
+        )
+        se_max_overall_duty_cycle: Final = ZCLAttributeDef(
+            id=0xE207,
+            type=t.uint16_t,
+            access="rw",
+            is_manufacturer_specific=True,
+        )
+        se_overall_duty_cycle_period: Final = ZCLAttributeDef(
+            id=0xE208,
+            type=t.uint16_t,
+            access="rw",
+            is_manufacturer_specific=True,
+        )
+        se_cluster_revision: Final = ZCLAttributeDef(
+            id=0xFFFD,
+            type=t.uint16_t,
+            access="r",
+            is_manufacturer_specific=True,
+        )
+
+
+(
+    QuirkBuilder(SE_MANUF_NAME, "EKO07259")
+    .applies_to(SE_MANUF_NAME, "WDE002497")
+    .applies_to(SE_MANUF_NAME, "WDE011680")
+    .replaces(SEBasic)
+    .replaces(SEBasic, endpoint_id=2)
+    .replaces(SEBasic, endpoint_id=3)
+    .replaces(SEBasic, endpoint_id=5)
+    .replaces(SEThermostat)
+    .replaces(SEUserInterface)
+    .replaces(SETemperatureMeasurement, endpoint_id=2)
+    .replaces(SETemperatureMeasurementExternal, endpoint_id=3)
+    .replaces(SEMetering, endpoint_id=5)
+    .adds(SECycleTime)
+    .adds(SEHeatingCoolingOutput)
```

Comment:
If these are also in the original signature (with no quirk), we should use `replaces` here too.